packages/Datepicker/src/components/Calendar.tsx

```diff
@@ -1,71 +1,72 @@
-import ChevronLeft from '@igloo-ui/icons/dist/ChevronLeft';
+    GregorianCalendar,
+    type Calendar as CustomCalendar
+} from "@internationalized/date";
+import ChevronRight from "@igloo-ui/icons/dist/ChevronRight";
+import ChevronLeft from "@igloo-ui/icons/dist/ChevronLeft";
 
-import CalendarButton from './CalendarButton';
-import CalendarGrid from './CalendarGrid';
+import CalendarButton from "./CalendarButton";
+import CalendarGrid from "./CalendarGrid";
 
 function createCalendar(identifier: string): CustomCalendar {
-  switch (identifier) {
-    case 'gregory':
-      return new GregorianCalendar();
-    default:
-      throw new Error(`Unsupported calendar ${identifier}`);
-  }
+    switch (identifier) {
+        case "gregory":
+            return new GregorianCalendar();
+        default:
+            throw new Error(`Unsupported calendar ${identifier}`);
+    }
 }
 
 interface CalendarProps extends AriaCalendarProps<DateValue> {
-  className?: string;
-  dataTest?: string;
-  highlightToday?: boolean;
+    className?: string;
+    dataTest?: string;
+    highlightToday?: boolean;
 }
 
 const Calendar: React.FunctionComponent<CalendarProps> = (
-  props: CalendarProps,
+    props: CalendarProps
 ) => {
-  const { className, dataTest, highlightToday } = props;
-  const { locale } = useLocale();
-  const state = useCalendarState({
-    ...props,
-    locale,
-    createCalendar,
-  });
+    const { className, dataTest, highlightToday } = props;
+    const { locale } = useLocale();
 
-  const ref = React.useRef<HTMLDivElement>(null);
-  const { calendarProps, prevButtonProps, nextButtonProps, title } =
+    const state = useCalendarState({
+        ...props,
+        locale,
+        createCalendar
+    });
+
+    const ref = React.useRef<HTMLDivElement>(null);
+    const { calendarProps, prevButtonProps, nextButtonProps, title } =
     useCalendar(props, state);
 
-  return (
-    <div
-      {...calendarProps}
-      ref={ref}
-      className={className}
-      data-test={dataTest}
-    >
-      <div className="ids-datepicker__header">
-        <h2 className="ids-datepicker__title">{title}</h2>
-        <CalendarButton {...prevButtonProps} className="ids-datepicker__prev">
-          <ChevronLeft size="medium" />
-        </CalendarButton>
-        <CalendarButton {...nextButtonProps} className="ids-datepicker__next">
-          <ChevronRight size="medium" />
-        </CalendarButton>
-      </div>
-      <CalendarGrid state={state} highlightToday={highlightToday} />
-    </div>
-  );
+    return (
+        <div
+            {...calendarProps}
+            ref={ref}
+            className={className}
+            data-test={dataTest}
+        >
+            <div className="ids-datepicker__header">
+                <h2 className="ids-datepicker__title">{title}</h2>
+                <CalendarButton {...prevButtonProps} className="ids-datepicker__prev">
+                    <ChevronLeft size="medium" />
```

Comment:
Is the size needed?

Reply:
yes. we get it from here which always adds a size https://igloo.officevibe.design/icons?search=%3DChevronLeft&size=medium
but I think the default size is small.

Reply:
Alright I am used with Orbit where medium is the default size.